src/NexusMods.App.UI/LeftMenu/Items/LeftMenuItemViewModel.cs

```diff
@@ -14,7 +15,7 @@ namespace NexusMods.App.UI.LeftMenu.Items;
 
 public class LeftMenuItemViewModel : AViewModel<ILeftMenuItemViewModel>, ILeftMenuItemViewModel
 {
-    [Reactive] public string Text { get; set; } = "";
+    [Reactive] public StringComponent Text { get; set; } = new("");
```

Comment:
This shouldn't be reactive but instead passed into the constructor.

Reply:
Good point, didn't change that after switching from normal text.

To avoid having to change all the code I made it Init with a default though, is that ok?